forwards_paper.tex

```diff
@@ -148,71 +133,99 @@ \section*{Introduction}
 so we discard all information irrelevant to the genetic history
 of the \emph{sampled} individuals,
 which results in considerable savings.
-
 Another way of representing this same information
 is known as the \emph{ancestral recombination graph}, or {ARG} \citep{griffiths1997ancestral},
 which has been the subject of substantial study
 under the assumptions of coalescent theory \citep{wiuf1997number,wiuf1999ancestry,marjoram2006coalescent,wilton2015smc}.
 
+\plr{Quick review of simulation software, esp that other one that does the same thing.}
+
+\plr{Make clear that the main product here is the \msprime{} interface,}
+part of which is the tables schema.
```

Comment:
need to explain what it is that this paper does.  and it's several things, I think:
 - explains some additions to msprime (tables format, simplify)
 - lays out forwards-recording
 - provides two implementations of this forwards-recording and benchmarks them

... ?

Reply:
This sounds right to me.